tests/integration-tests/tests/schedulers/test_slurm.py

```diff
@@ -1537,6 +1579,21 @@ def _test_recover_from_protected_mode(
     check_status(cluster, compute_fleet_status="RUNNING")
 
 
+@retry(wait_fixed=seconds(30), stop_max_delay=minutes(20))
+def _test_compute_fleet_status(command_executor, expected_status):
+    """Assert that the compute fleet status is the expected status."""
+    result = command_executor.run_remote_command("get-compute-fleet-status.sh").stdout
+    status = json.loads(result)
+    assert_that(status.get("status")).is_equal_to(expected_status)
+
+
+@retry(wait_fixed=seconds(10), stop_max_delay=minutes(5))
+def _test_cluster_creation_failure(cluster, failure_code="HeadNodeBootstrapFailure"):
+    """Retry describe-cluster until we see {failure_code}."""
+    response = cluster.describe_cluster()
+    assert_that(response["failures"][0]["failureCode"]).is_equal_to(failure_code)
+
```

Comment:
Could you add a check for the logs in the chef-client logs related to protected mode during static fleet provisioning?

Reply:
Added in an assertion in the main test function.